src/Nerve_GIC/include/gudhi/GIC.h

```diff
@@ -167,14 +163,6 @@ class Cover_complex {
   // Utils.
   // *******************************************************************************************************************
 
- public:
-  /** \brief Specifies whether the type of the output simplicial complex.
-   *
-   * @param[in] t std::string (either "GIC" or "Nerve").
-   *
-   */
-  void set_type(const std::string& t) { type = t; }
```

Comment:
In src/Nerve_GIC/utilities/Nerve.cpp, `SC.set_type("Nerve");` does not compile.
In src/Nerve_GIC/utilities/VoronoiGIC.cpp, `GIC.set_type("GIC");` does not compile.
In src/Nerve_GIC/test/test_GIC.cpp, `N.set_type("Nerve");` and `GIC.set_type("GIC");` does not compile.
In src/Nerve_GIC/example/FuncGIC.cpp, `GIC.set_type("GIC");` does not compile.
In src/Nerve_GIC/example/CoordGIC.cpp, `GIC.set_type("GIC");` does not compile.
In src/python/doc/nerve_gic_complex_user.rst, all the `nerve_complex.set_type('...')` fail when generating the documentation.
In src/python/example/coordinate_graph_induced_complex.py, `nerve_complex.set_type("GIC")` fails.
In src/python/example/functionnal_graph_induced_complex.py, `nerve_complex.set_type("GIC")` fails.
In src/python/example/nerve_of_a_covering.py, `nerve_complex.set_type("GIC")` fails.
In src/python/example/voronoi_graph_induced_complex.py, `nerve_complex.set_type("GIC")` fails.
In src/python/test/test_cover_complex.py, `.set_type("...")` fails.

Reply:
Should work now that I put Nerve back